invenio_rdm_records/resources/serializers/__init__.py

```diff
@@ -8,12 +8,13 @@
 
 """Record response serializers."""
 
-from .csl import CSLJSONSerializer
+from .csl import CSLJSONSerializer, StringCitationSerializer
 from .datacite import DataCite43JSONSerializer
 from .ui import UIJSONSerializer
 
 __all__ = (
     "UIJSONSerializer",
+    "StringCitationSerializer",
     "CSLJSONSerializer",
     "DataCite43JSONSerializer",
```

Comment:
I know the UI was already there... But could you changed them and sort them alphabetically?
```suggestion
    "CSLJSONSerializer",
    "DataCite43JSONSerializer",
    "StringCitationSerializer",
    "UIJSONSerializer",
```